plugins/source/k8s/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.19
 require (
 	github.com/cloudquery/plugin-sdk v1.5.2
 	github.com/golang/mock v1.6.0
+	github.com/google/gnostic v0.5.7-v3refs
```

Comment:
This k8s change seems unrelated to this PR, can we do it in another one?

Reply:
we can, I just ran `go mod tidy` in all of our plugins

Reply:
Got it, but if we merge this PR a new changelog entry for k8s will be created with `Update to new AWS SDK` which is a bit confusing

Reply:
I see that you reverted it, thanks!